tendrl/node_agent/message/handler.py

```diff
@@ -45,7 +45,7 @@ def read_socket(self, sock, *args):
                         message
                     )
             Logger(message)
-        except (socket.error, socket.timeout):
+        except(socket.error, socket.timeout, RuntimeError):
```

Comment:
we are raising RunTimeError here but it is not handled anywhere https://github.com/Tendrl/node-agent/pull/846/files#diff-3eae70b9dbe53a810f8c236acc9baa6dL65.


This is verified